packages/writing-style/bin/commercetools-vale.js

```diff
@@ -8,7 +8,16 @@ const binaryFileName = os.platform() === 'win32' ? 'vale.exe' : 'vale';
 
 const valeBinary = path.join(__dirname, `./${binaryFileName}`);
 const valeConfig = path.join(__dirname, `../.vale.ini`);
-const result = shelljs.exec(
-  `${valeBinary} --config ${valeConfig} ${process.argv.slice(2).join(' ')}`
```

Comment:
Out of curiosity: what happens if we pass the `--config` as last parameter? Will that be used even if there are duplicated params before that?

```
`${valeBinary} ${process.argv.slice(2).join(' ')} --config ${valeConfig}`
```

Reply:
It doesn't catch that case but as said the implementation is specifically targeting the situation with the VSCode plugin which passes the --format parameter after an empty --config.    

passing --config as last or second last does not work in any case (also directly against the vale binary).  You need to pass a config file after it and then the directory to lint at the end.